jedeschule/pipelines/school_pipeline.py

```diff
@@ -104,17 +111,26 @@ def process_item(self, item, spider):
                             phone=item.get('Telefon'),
                             )
         elif spider.name == 'brandenburg':
-            school = School(
```

Comment:
Please don't change the indentation for the `Brandenburg` section here. Adding Rheinland-Pfalz should be enough